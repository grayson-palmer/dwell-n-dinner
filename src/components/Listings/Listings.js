import React, { Component } from 'react';
import './Listings.scss';

export default Listings = () => {
  return ()
}