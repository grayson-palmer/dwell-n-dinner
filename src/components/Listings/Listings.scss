.listing-section {
  display: flex;
  flex-flow: row wrap;
  justify-content: center;
  align-items: center;
  .listing-card {
    border: .5rem solid black;
    border-radius: 1rem;
    width: 25em;
    margin: 1em;
  }
  .listing-image {
    width: 100%;
    & > img {
      object-fit: contain;
    }
  }
}
