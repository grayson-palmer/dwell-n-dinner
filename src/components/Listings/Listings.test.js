import React from 'react';
import { shallow } from 'enzyme';
import Listings from './Listings';

describe('Listings', () => {
  it('should render the component', () => {
    const wrapper = shallow(
      <Listings
        listings={[
          {
            name: 'House_1',
            listing_id: '3',
            area_id: '590'
          }
        ]}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
