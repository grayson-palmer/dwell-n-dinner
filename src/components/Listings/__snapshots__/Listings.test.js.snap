// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Listings should render the component 1`] = `
<section
  className="listings"
/>
`;
